<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Dech-PC" ignored="0" name="Firefox Test" tests="1" failures="1" timestamp="01 Jun 2021 14:25:32 GMT" time="39.136" errors="0">
  <testcase name="test" time="32.346" classname="test.TestNG_TestCase">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at test.TestNG_TestCase.test(TestNG_TestCase.java:78)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- test -->
</testsuite> <!-- Firefox Test -->
